soh/soh/Enhancements/randomizer/randomizer_entrance_tracker.cpp

```diff
@@ -797,6 +805,11 @@ void EntranceTrackerWindow::DrawElement() {
                     continue;
             }
 
+            // RANDOTODO: Only show blue warps if bluewarp shuffle is on
+            if (original->metaTag.ends_with("bw") || override->metaTag.ends_with("bw")) {
+                continue;
+            }
```

Comment:
out of scope for this PR (and i don't know if this is handled differently in rando v3), but this feels like a perfect example of the fragility of string based metatags (`ENTR_LAKE_HYLIA_9` wouldn't work if the tag was entered as `"bw,lh"` instead of `"lh,bw"`)

depending on how many tags we have it might make sense to use bitpacking instead